src/pretix/plugins/sendmail/views.py

```diff
@@ -95,3 +120,23 @@ def get_context_data(self, *args, **kwargs):
         ctx = super().get_context_data(*args, **kwargs)
         ctx['output'] = getattr(self, 'output', None)
         return ctx
+
+
+class EmailHistoryView(EventPermissionRequiredMixin, ListView):
+    template_name = 'pretixplugins/sendmail/history.html'
+    permission = 'can_change_orders'
+    model = LogEntry
```

Comment:
If you add ``context_object_name = 'logs'`` here, you can just reference the list in the template as ``logs``. If not, it *probably* is ``logentry_list``, but let's better be explicit about it :)